src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -15,7 +15,7 @@
             + "provided is invalid";
     public static final String MESSAGE_MORE_THAN_ONE_SAME_PREFIX = "There should only be 1 option";
     public static final String MESSAGE_DUPLICATE_APPOINTMENT =
-            "You already have an appointment with the same name/date in the address book.";
+            "You already have an appointment with the same name/date/address in the address book.";
```

Comment:
It's an "and" relationship if I remembered correctly, so maybe "with the exact same name, date and address" would be better?

Reply:
Okay! I'll update it accordingly.

Reply:
Okay 😄 👍